docs/README.md

```diff
@@ -15,10 +15,10 @@ This repository contains the ToolJet documentation website code and Markdown sou
 - [Local setup](#local-setup)
 
 ## Feedback
-If you want to give documentation feedback, please join our [Slack Community](https://tooljet.com/slack) and drop us a message.
+If you would like to provide feedback on the documentation, feel free to join our [Slack Community](https://tooljet.com/slack) and send us a message.
 
 ## Documentation Issues
-To enter documentation bugs or submit any feature request for documentation, please create a new [GitHub issue](https://github.com/ToolJet/ToolJet/issues/new?assignees=&labels=documentation&template=03_documentation_report.yml&title=%5Bdocs%5D%3A+). Please check if there is an existing issue first.
+If you encounter any issues with the documentation or have a feature request related to it, you can create a new [GitHub issue](https://github.com/ToolJet/ToolJet/issues/new?assignees=&labels=documentation&template=03_documentation_report.yml&title=%5Bdocs%5D%3A+) using the provided template. Please ensure that you check for existing issues before creating a new one.
```

Comment:
If you come across any issues with the documentation or have a feature request related explicitly to it, we encourage you to create a new [GitHub issue](https://github.com/ToolJet/ToolJet/issues/new?assignees=&labels=documentation&template=03_documentation_report.yml&title=%5Bdocs%5D%3A+) following the template. Before creating a new issue, we kindly request that you check for existing issues to avoid duplication.